presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java

```diff
@@ -4701,9 +4701,9 @@ public void testAccessControl()
         assertAccessAllowed("SHOW STATS FOR lineitem", privilege("orders", SELECT_COLUMN));
         assertAccessAllowed("SHOW STATS FOR (SELECT * FROM lineitem)");
         assertAccessAllowed("SHOW STATS FOR (SELECT * FROM lineitem)", privilege("orders", SELECT_COLUMN));
-        assertAccessDenied("SHOW STATS FOR (SELECT * FROM lineitem)", "Cannot select from columns \\[.*\\] in table or view .*.lineitem.*", privilege("lineitem", SELECT_COLUMN));
```

Comment:
Would be good to remove `.*` in the commit where you added this test, so that the next commit shows what's being changed here

Also, add test cases where `lineitem` is qualified with schema and catalog.schema.

@kokosing @electrum  what about moving `io.prestosql.testing.AbstractTestQueries#testAccessControl` to TLQ? Maybe along with `testNonQueryAccessControl`?

Reply:
> io.prestosql.testing.AbstractTestQueries#testAccessControl to TLQ? Maybe along with testNonQueryAccessControl?

I think we should extract it to `TestAccessControl` (not yet existing) where would have custom plugin (connector) used so we could test not only table access but also access to other stuff too (like queries, impersonation, session properties and futures, procedures in future).